frontend/src/pull-card/flags.tsx

```diff
@@ -18,6 +18,7 @@ export const Flags = memo(function Flags({ pull }: { pull: Pull }) {
   const devBlock = pull.getDevBlock();
   const draftBlock = pull.isDraft();
   const readyButNoCI = pull.isReady() && !pull.isCiRequired();
+  const dependentPR = pull.isDependent() && !pull.getDeployBlock() && pull.isCiRequired() && !pull.isDraft();
```

Comment:
I might just use `isDependent()` here cause I think the flag is useful no matter the deploy state.

Reply:
Should I maybe add a new FA icon / color change for this flag?

The reason being is that when it's just isDependent() then there are duplicate flags

e.g. 

![Screenshot 2023-03-17 121430](https://user-images.githubusercontent.com/95656772/225998632-0b5b716d-7086-4690-8c60-0cec95a10a7d.jpg)

So I opted to prioritize the deploy_block label first, or the no CI flag

Reply:
I think a different icon is appropriate: https://fontawesome.com/search?q=arrow&o=r&m=free&s=solid

Maybe:
![image](https://user-images.githubusercontent.com/26855/226000723-fda8caf1-0cd6-4e7b-8c0d-1a28f04d7af5.png)

Or...   something else?

Reply:
https://fontawesome.com/icons/code-branch?s=solid

Reply:
Good choice.